src/desktop/apps/artwork/components/commercial/templates/acquire.jade

```diff
@@ -1,6 +1,15 @@
-if artwork.is_acquireable
-  .artwork-acquire-form
+.artwork-acquire-form
     input( type='hidden', name='artwork_id', value= artwork.id )
 
-    button.avant-garde-button-black.is-block( class='js-artwork-acquire-button' )
-      | Buy now
+    if artwork.is_acquireable && artwork.is_offerable
+      button.avant-garde-button-black.is-block( class='js-artwork-acquire-button' )
+        | Buy now
+      
+      button.avant-garde-button-white.artwork-button-second-action.is-block( class='js-artwork-offer-button' )
+        | Make offer
+    else if artwork.is_acquireable
+      button.avant-garde-button-black.is-block( class='js-artwork-acquire-button' )
+        | Buy now
+    else
```

Comment:
Should this be `else if artwork.is_offerable` instead? Otherwise an artwork that's not acquireable or offerable could still show a make offer button.

Reply:
This template only gets included if `if artwork.is_acquireable || artwork.is_offerable` (below)

Reply:
Ah okay, I see that you also removed the check for inquireable works in `inquire.jade`, too. I'm mostly thinking about whether or not having this template implicitly assume a particular artwork state could produce bugs down the road, but if it's only used here I'm 👍